File: pinyougou-interface/src/main/java/com/pinyougou/core/service/AddressService.java
package com.pinyougou.core.service;

import cn.itcast.core.pojo.address.Address;

import java.util.List;

public interface AddressService {

    /**
     * 获取当前登录人的收货地址列表
     * @param name
     * @return
     */
    public List<Address> findListByLoginUser(String name);
}

File: pinyougou-web-user/src/main/webapp/js/service/addressService.js
//服务层
app.service('addressService',function($http){

    //获取当前登录账号的收货地址
    this.findAddressList=function(){
        return $http.post('../address/findListByLoginUser.do');
    }

    //获取地区
    this.findByParentId = function(parentId){
        return $http.get("../address/findByParentId.do?parentId="+parentId);
    }

});